lms/djangoapps/instructor/views/api.py

```diff
@@ -1735,21 +1779,21 @@ def get_student_progress_url(request, course_id):
         'progress_url': '/../...'
     }
     """
-    course_id = CourseKey.from_string(course_id)
-    user = get_student_from_identifier(request.POST.get('unique_student_identifier'))
+    authentication_classes = (JwtAuthentication, BearerAuthentication, SessionAuthentication)
```

Comment:
I think you don't need to define this, since the base authentication classes should be the correct ones.

Reply:
Now I simply matched the other api in this file https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/instructor/views/api.py#L1587